dev/generate_traffic.py

```diff
@@ -0,0 +1,151 @@
+import requests
+import argparse
+import threading
+import time
+import json 
+import random 
+import uuid
+
+global url
+global keycloak
+global username
+global password
+global bearer_token
+
+bridges = []
+processors = []
+
+def create_bridge():
+    while True:
+        if (random.random() < 0.2 and bridges): # with probability 0.2 create a bad request
```

Comment:
What about moving the value `0.2` (and the others of this file) as optional input parameters?